js/test.js

```diff
@@ -36,72 +25,93 @@ if (window.location.pathname.endsWith ('quiz.html')) {
   } else {
     alert('Your answer must be a yes or no.');
   }
+ }
+ quizHoodFunc()
```

Comment:
Missing semi colon here and on other functions you call. Check your linter errors for more.